core/client/src/main/java/alluxio/client/file/policy/LocalFirstAvoidEvictionPolicy.java

```diff
@@ -32,8 +33,10 @@
  * to store the block, for the values mCapacityBytes minus mUsedBytes is not the available bytes.
  */
 @ThreadSafe
-public final class LocalFirstAvoidEvictionPolicy implements FileWriteLocationPolicy {
+public final class LocalFirstAvoidEvictionPolicy
```

Comment:
We are going to deprecate FileWriteLocationPolicy in 2.0. At that time, all of these will be moved to block.policy. If you prefer to duplicating the code in file.policy and block.policy before 2.0, i can do that in another PR just to in case some other people have different suggestion. Does that sound good to you?